src/index.js

```diff
@@ -235,7 +243,7 @@ document.addEventListener("DOMContentLoaded", () => {
   } else {
     solrClient.initialize();
   }
-
+  fetchVocabulary();
```

Comment:
This seems to be taking much longer than I would expect.  I am probably doing it incorrectly…